### DISCUSSION TITLE
logical interpretations

### DISCUSSION THREAD

**Discussion by arademaker** [delph-in/docs#134] [Q&A]
> he does not run fast

What reading do you prefer?

1. ∃ x3, pron x3 ∧ ¬(∃ e2 e11, _run_v_1 e2 x3 ∧ _fast_a_1 e11 e2)
2. ∃ x3, pron x3 ∧ (∀ e10, , _run_v_1 e10 x3 → ¬ ∃ e11, _fast_a_1 e11 e10)
3. ∃ x3, pron x3 ∧ ¬ ∃ e13,_fast_a_1 e13 x3 ∧ ∃ e10, , _run_v_1 e10 x3

Formulas 2 and 3 can't be produced from the readings I got.

BTW, I just found a mistake in my translations when dealing with the reading where fast modify the negation:

```
[ TOP: h0
  INDEX: e2
  RELS: < [ pron<0:2> LBL: h4 ARG0: x3 ]
          [ pronoun_q<0:2> LBL: h5 ARG0: x3 RSTR: h6 BODY: h7 ]
          [ neg<8:11> LBL: h1 ARG0: e2 ARG1: h8 ]
          [ _run_v_1<12:15> LBL: h9 ARG0: e10 ARG1: x3 ]
          [ _fast_a_1<16:20> LBL: h1 ARG0: e11 ARG1: e2 ] >
  HCONS: < h0 qeq h1 h6 qeq h4 h8 qeq h9 > ]

 (0/2) ∃ e2 e11, ¬(∃ x3, pron x3 ∧ (∃ e10, _run_v_1 e10 x3)) ∧ _fast_a_1 e11 e2
 (1/2) ∃ e2 x3, pron x3 ∧ (∃ e11, ¬(∃ e10, _run_v_1 e10 x3) ∧ _fast_a_1 e11 e2)
```

**Comment by guyemerson** [comment]
I think 1 is right.

It seems 2 is logically equivalent to 1.  But 1 is what would be produced compositionally.

3 is logically stronger, asserting that he is running but not running fast.  This might be plausible pragmatically, if the focus of negation is on "fast".  But I don't think this should be part of the semantics.
